crates/sui-protocol-config/src/lib.rs

```diff
@@ -264,14 +267,25 @@ struct FeatureFlags {
 
     #[serde(skip_serializing_if = "is_false")]
     enable_jwk_consensus_updates: bool,
+
+    // Max number of publish or upgrade commands allowed in a
+    // programmable transaction block.
+    #[serde(skip_serializing_if = "is_zero")]
+    max_publish_or_upgrade_per_ptb: u64,
```

Comment:
This should be in the protocol config attributes, not feature flags.
Hence `max_publish_or_upgrade_per_ptb: Option<u64>`
`max_publish_or_upgrade_per_ptb` getter will be auto-derived as well so no need to impl it explicitly on line 789.

Reply:
not too sure why you want it in the protocol config instead of a feature flag. 
Can you articulate a bit more?

Reply:
though it could make more sense in the protocol config, not sure, I'll review it.
Thoughts welcome anyway

Reply:
I move the flag to the regular protocol config, turns out I liked it more. Thanks!

Reply:
(Answering more for posterity than anything else)

> not too sure why you want it in the protocol config instead of a feature flag.
> Can you articulate a bit more?

I don't have all the context on why they were separated out from the rest of the protocol confi, but feature flags are a subset of the the protocol config which typically:

- only contain `bool`s,
- are only used to control whether a given feature is enabled or not, and
- are not turned off after they are turned on.

`max_publish_or_upgrade_per_ptb` in a new limit, rather than a feature flag -- it's not a `bool`, it's imposing a new limit rather than gating a new feature, and we may tweak this number multiple times in future.